dbcon/joblist/tuple-bps.cpp

```diff
@@ -1691,9 +1703,13 @@ bool TupleBPS::processPseudoColFilters(uint32_t extentIndex, boost::shared_ptr<m
                && (!hasSegmentDirFilter || processOneFilterType(8, emEntry.partitionNum, PSEUDO_SEGMENTDIR))
                && (!hasExtentIDFilter || processOneFilterType(8, emEntry.range.start, PSEUDO_EXTENTID))
                && (!hasMaxFilter || (emEntry.partition.cprange.isValid == BRM::CP_VALID ?
-                                     processOneFilterType(emEntry.range.size, emEntry.partition.cprange.hi_val, PSEUDO_EXTENTMAX) : true))
+                                     (!datatypes::Decimal::isWideDecimalType(fColType) ?
```

Comment:
Who is the author of this. I wish him to work with such condition expressions all his life.

Reply:
That's a million dollar question you have asked, I wish I had the answer. But I take part of the blame :)